<?xml version="1.0"?>
<launch>
    <node pkg="stefmap_ros" type="stefmap_node_offline.py" name="stefmap_node" output="screen">
        <param name="grid_size" value="1"/>  
        <param name="x_min" value="-10" />
        <param name="x_max" value="60" />
        <param name="y_min" value="-10" />
        <param name="y_max" value="60" />
        <param name="num_bins" value="8" />
        <param name="frame_id" value="map_laser2d" />
    </node>
    
    <node pkg="fremenarray" type="fremenarray" name="fremenarray" output="screen">
    </node>

<!--
    <node pkg="stefmap_ros" type="stefmap_client_example.py" name="stefmap_client_node" output="screen" >
    </node>

    <node pkg="tf" type="static_transform_publisher" name="world_to_map" args="-10.5 -27 0 0.2 0 0 /world /stefmap 100" >
    </node>
-->


</launch>